src/ethereum/networkAdapters/EvmScanAdapter.ts

```diff
@@ -557,6 +567,11 @@ export function mergeEdgeTransactions(
       isSend: nativeAmount.startsWith('-'),
       nativeAmount,
       networkFee: mergedNetworkFee,
+      ...(mergedParentNetworkFee != null
```

Comment:
I get it. but this is a good example of high cognitive load to understand. Maybe in the future declare the tx object outside of the `.set()`, then add the `parentNetworkFee` to it if needed. Then call the .set() with the tx object

```
const edgeTransaction: EdgeTransaction =  {
      ...existingTransaction,
      isSend: nativeAmount.startsWith('-'),
      nativeAmount,
      networkFee: mergedNetworkFee,
      ourReceiveAddresses: [
        ...existingTransaction.ourReceiveAddresses,
        ...transaction.ourReceiveAddresses
      ]
}

if (mergedParentNetworkFee != null) {
  edgeTransaction.parentNetworkFee = mergedParentNetworkFee
}

txidToTransaction.set(uniqueKey, edgeTransaction)
```